litmus-portal/graphql-server/manifests/namespace/3a_agents_rbac.yaml

```diff
@@ -29,7 +29,7 @@ rules:
 
   - apiGroups: ["apps"]
     resources: ["deployments", "daemonsets", "replicasets", "statefulsets"]
-    verbs: ["get", "list", "delete"]
+    verbs: ["get", "list", "delete","deletecollection"]
```

Comment:
And here for namespace scope - 
https://github.com/litmuschaos/litmus/blob/ce5b7dd806eed34451f85607daed43ef5c14331a/litmus-portal/manifests/namespace-k8s-manifest.yml#L149